app/questionnaire/questionnaire_schema.py

```diff
@@ -161,24 +161,30 @@ def _get_answers_by_id(self) -> dict[str, list[ImmutableDict]]:
 
         return answers_by_id
 
-    def get_hub(self) -> Optional[ImmutableDict]:
-        hub_schema: Optional[ImmutableDict] = self.json.get("hub")
-        return hub_schema
+    @cached_property
+    def _questionnaire_flow(self) -> ImmutableDict[str, Any]:
+        questionnaire_flow: ImmutableDict = self.json["questionnaire_flow"]
+        return questionnaire_flow
+
+    @cached_property
+    def questionnaire_flow_options(self) -> ImmutableDict[str, Any]:
+        options: ImmutableDict[str, Any] = self._questionnaire_flow["options"]
+        return options
 
-    def is_hub_enabled(self) -> Optional[bool]:
-        if hub := self.get_hub():
-            return hub.get("enabled")
+    @cached_property
+    def is_questionnaire_flow_hub(self) -> bool:
+        return bool(self._questionnaire_flow["type"] == "Hub")
```

Comment:
I think referencing `hub` is fine since that is the `type`. We could also have more than 2 types in future. I don't think calling it `non-linear` is clear, and I can't think of anything else that is better.

Reply:
It would be nice for them to both be adjectives, but it was more a observation rather than a change request